HDPS/src/private/PluginManager.cpp

```diff
@@ -532,20 +533,55 @@ void PluginManager::fromVariantMap(const QVariantMap& variantMap)
 
     if (!missingPluginKinds.isEmpty())
         throw std::runtime_error(QString("One or more plugins are not available: %1").arg(missingPluginKinds.join(", ")).toLocal8Bit());
+
+    for (const auto& loadedAnalysis : variantMap["LoadedAnalyses"].toList())
+    {
+        auto analysisPluginMap = loadedAnalysis.toMap();
+
+        variantMapMustContain(analysisPluginMap, "Kind");
+        variantMapMustContain(analysisPluginMap, "inputDatasetsGUIDs");
```

Comment:
I'd propose to change `"inputDatasetsGUIDs"` to `"InputDatasetsGUIDs"` since all `QVariantMap` keys in the core use title case.